### ISSUE TITLE
Colors in QGIS wasm in chromium look less bright (washed out)

### ISSUE THREAD

**Issue by andreasneumann** [qgis/qgis-js#9]
![grafik](https://github.com/qgis/qgis-js/assets/884476/42f10ca4-b7ce-44d0-a1d0-5016697a4a13)

It seems like Chrome does some color management which results in slightly dumper (less vivid) colors - when compared to QGIS Desktop and Firefox wasm rendering.

This is more an observation and not so much a bug.

**Comment by andreasneumann** [comment]
Maybe it has to do with the fact that browsers behave differently for images where no color profile is embedded in the image - see this comparison table at https://www.benq.com/en-us/knowledge-center/knowledge/web-browsers-color-management.html

Perhaps this can be fixed by assigning the sRGB color profile to the rendered canvas image?

**Comment by boardend** [comment]
Does setting the `chrome://flags/#force-color-profile` have an effect on this?

We can try to set `srgb` on the [ImageData](https://developer.mozilla.org/en-US/docs/Web/API/ImageData/colorSpace) returned form the qgis-js API, as well as in the [Canvas](https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement/getContext?retiredLocale=de#colorspace). (But this will be the default anyways, so in theory it should have no effect...)

PS: On my system (Ubuntu 22.04 with Wayland) the colors in Firefox and Chromium seem to look the same. I will have a go on some Windows machines over the weekend.

**Comment by andreasneumann** [comment]
> Does setting the `chrome://flags/#force-color-profile` have an effect on this?

Yes - there is a huge difference. Setting it to sRGB makes the colors the same as in QGIS and Firefox, setting this preference to "Default" results in the washed out colors that I described.

Does this mean that there is nothing we could do here - and it is up to the user setting ?

**Comment by boardend** [comment]
One can probably also disable [HDR in Windows](https://support.microsoft.com/en-us/windows/hdr-settings-in-windows-2d767185-38ec-7fdc-6f97-bbc6c5ef24e6) while leaving Chromiums color profile on "default"?

Could you also do a test, what the following snippet prints to the console, if you paste it to the dev tools of FF and Chromium?

```
console.log("SDR:", window.matchMedia("(dynamic-range: standard)").matches)
console.log("HDR:", window.matchMedia("(dynamic-range: high)").matches)
```

**Comment by andreasneumann** [comment]
Chrome 117.0.5938.150 (Official Build) (64-bit Windows):
```
console.log("SDR:", window.matchMedia("(dynamic-range: standard)").matches)
SDR: true
console.log("HDR:", window.matchMedia("(dynamic-range: high)").matches)
HDR: false
```

Firefox 117.0.1 (64-Bit Windows):
```
console.log("SDR:", window.matchMedia("(dynamic-range: standard)").matches)
SDR: true debugger eval code:1:9
console.log("HDR:", window.matchMedia("(dynamic-range: standard)").matches)
HDR: true debugger eval code:1:9
```
